src/features/auth/utils/constants.js

```diff
@@ -0,0 +1,7 @@
+export const FULL_NAME_MIN_LENGTH = 2;
+export const FULL_NAME_MAX_LENGTH = 50;
+
+export const USERNAME_MIN_LENGTH = 2;
+export const USERNAME_MAX_LENGTH = 50;
+
+export const PASSWORD_REGEX = new RegExp('^(?=.*[A-Za-z])(?=.*\\d)[A-Za-z\\d]{8,}$');
```

Comment:
I just improvised some placeholder constraints but I'm not sure if there's any definition about this.